### DISCUSSION TITLE
how to remove right and bottom padding on macos

### DISCUSSION THREAD

**Discussion by a-bit-digital-hk** [ghostty-org/ghostty#3716] [Q&A] [answered]
<img width="855" alt="SCR-20241228-ryii" src="https://github.com/user-attachments/assets/446fd99d-0f81-4c61-adb7-b0c714c4255f" />
this is my config
macos-titlebar-style = hidden
window-decoration = false

font-size = 20
font-thicken = true

theme = BlueBerryPie
window-padding-color = background
window-padding-x = 0
window-padding-y = 0

i want to remove the the padding on the right side and bottom side. please help. thanks

**Comment by eikopf** [comment]
As far as I'm aware this is a neovim issue; have a look at your options for `cmdheight`, `cmdwinheight`, and `shortmess` (possibly also `laststatus` and `showcmd`).

**Comment by 00-kat** [comment, marked as answer]
That padding is there because your window size is not a multiple of the cell size.  Try [`window-padding-balance`](https://ghostty.org/docs/config/reference#window-padding-balance) and [`window-padding-color`](https://ghostty.org/docs/config/reference#window-padding-color).

**Comment by a-bit-digital-hk** [comment]
solved thanks

**Reply by hoangtrucit** [reply]
@a-bit-digital-hk could you share your full config? I also face same issue but I can't fix it. Thanks

**Reply by anaetrezve** [reply]
I am using Macbook Pro M1 with  Resolution:`1680x1050, 1920x1080` and I was able to solve this issue with the below config

```
theme = catppuccin-mocha
font-family = CartographCF Nerd Font 
font-size = 17
font-synthetic-style = true
adjust-cell-height = 47%

cursor-style = block
cursor-style-blink = true
shell-integration-features = no-cursor

fullscreen = true
maximize = true

window-padding-balance = true
window-padding-x = 0
window-padding-y = 0
```

**Comment by CRAG666** [comment]
@00-kat Not working for me
![image](https://github.com/user-attachments/assets/817e481c-eed1-4ab5-98be-14784b6ca97c)
config:
shell-integration = zsh
font-family = "DankMono Nerd Font Mono"
# font-family = "Maple Mono NF CN"
font-synthetic-style = true
font-size = 15
shell-integration-features = no-cursor,sudo,no-title
mouse-hide-while-typing = true
mouse-scroll-multiplier = 2
theme = catppuccin-mocha
window-decoration = false
window-padding-x = 0
window-padding-y = 0
window-padding-balance = true
window-padding-color = background

**Reply by 00-kat** [reply]
`window-padding-color = background` is the default, I'm not sure what you expected there.  Try `window-padding-color = extend`.

**Comment by ibehnam** [comment]
still not working:

Ghostty:

<img width="1352" alt="image" src="https://github.com/user-attachments/assets/451578fb-7ded-4c3e-8953-f2c6b29cc7e1" />


iterm2 (same tmux session):

<img width="1352" alt="image" src="https://github.com/user-attachments/assets/264d8cc9-2ba8-48f2-bb8f-fe5afbf7d844" />

My ghostty config:



```
title = " "
window-padding-balance = true
window-padding-color = extend
window-save-state = always
macos-titlebar-style = "transparent"
window-colorspace = "display-p3"
background = 1C2021
foreground = d4be98
background-blur-radius = 20
```

**Comment by datacurse** [comment]
thanks everyone for the thread! this helped me:

```toml
window-padding-balance = true
window-padding-x = 0
window-padding-y = 0
window-padding-color = extend
```

**Reply by 00-kat** [reply]
@gabrielh-muniz in the future, consider using the upvote button or 👍️ reaction instead; it avoids sending an email to all subscribers.

**Comment by vuthanhtung2412** [comment]
In addition to 
```
window-padding-balance = true
window-padding-x = 0
window-padding-y = 0
```

playing around with `adjust-cell-height = 47%`, should fix the problem
